subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/eclipse/EclipseJavaProjectModulesIntegrationTest.groovy

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.plugins.ide.eclipse
+
+import org.gradle.integtests.fixtures.ToBeFixedForConfigurationCache
+import org.gradle.util.Requires;
+import org.gradle.util.TestPrecondition
+
+@Requires(TestPrecondition.JDK9_OR_LATER)
+class EclipseJavaProjectModulesIntegrationTest extends AbstractEclipseIntegrationSpec {
+
+    @ToBeFixedForConfigurationCache
```

Comment:
@donat 
When is this necessary?
@CsCherrYY 
Did you run into Problems without it?

Reply:
When the plugin or test is not configuration cache compatible. More concretely, when the `configurationCacheIntegTest` task fails and the `integTest` task passes.

Reply:
I didn't try this. And just follow the convention from other integ test cases. If it's useless, I would remove it.

Reply:
@CsCherrYY I checked this can be removed.
Please do that.
thx.

Reply:
sure. done in https://github.com/gradle/gradle/pull/24628/commits/75f13cec8e9f0131bbddd9a6f57991d8ea4fe09c